javascript/widgets/config.js

```diff
@@ -290,6 +290,24 @@ firebaseui.auth.widget.Config.prototype.getTosUrl = function() {
 };
 
 
+/**
+ * @return {boolean} Whether the display name should be displayed.
+ * Defaults to true.
+ */
+firebaseui.auth.widget.Config.prototype.getRequireDisplayName = function() {
+  var signInOptions = this.getSignInOptions_();
+  var requireDisplayName = true;
+
+  for (var i = 0; i < signInOptions.length; i++) {
+    if (signInOptions[i]['provider'] === firebaseui.auth.idp.EMAIL_PROVIDER_ID
```

Comment:
Tried the same but got a `firebase is not known` error in the tests. Didn't see any other test importing `firebase.auth` so wasn't sure if its safe to add that dependancy.